src/Forms/Checkbox.js

```diff
@@ -77,7 +83,7 @@ const Checkbox = React.forwardRef(({
                 className={labelClasses}
                 disabled={disabled}
                 htmlFor={checkId}>
-                {children}
+                {checkboxChildren}
```

Comment:
on a related note, is children a required prop, I saw this warning in our Table stories
```
Warning: Failed prop type: The prop `children` is marked as required in `Checkbox`, but its value is `undefined`.
    in Checkbox (created by storyFn)
    in storyFn
    in ErrorBoundary
```

Reply:
I think it was me that set it to required - from my memory it was to enforce the checkbox having a label

Reply:
Unless they change the design for the table with checkbox, we need to mark `children` as not required and provide an aria-label alternative. Or do a custom prop so we require either `children` or an `aria-label`.